node/src/blockchain/blockchain_interface/blockchain_interface_null/mod.rs

```diff
@@ -182,15 +184,20 @@ mod tests {
         let wallet = make_wallet("blah");
         let persistent_config = PersistentConfigurationMock::new();
 
-        let result = make_subject(test_name)
-            .build_blockchain_agent(&wallet, &persistent_config)
-            .unwrap();
+        let result = make_subject(test_name).build_blockchain_agent(&wallet, &persistent_config);
```

Comment:
You thought maybe you should have `subject` as its own variable. I have an opinion, but I'm leaving it up to you.